### PR TITLE
Make all component DOIs optional

### PR DESCRIPTION
**Pull request by thewilkybarkid**
https://github.com/elifesciences/issues/issues/4781

### CODE REVIEW

**Review by gnott — APPROVED** [review]
Looks right to me - thanks!

File: src/api.raml
Comment (lsh-0) [inline src/api.raml:464]: are these two articles (569, 15278) special in some way or just plain examples of actual articles?

**Review by lsh-0 — COMMENTED** [review]
I'll defer to @thewilkybarkid and @gnott on the correctness of the changes and raise any questions in the related ticket here: https://github.com/elifesciences/issues/issues/4781

**Comment by lsh-0** [comment]
dumb question: the related ticket is about making DOIs optional, but the title of this PR is about making component DOIs (10.7554/eLife.09560.**001**) optional. Is it one or the other or both?

**Comment by thewilkybarkid** [comment]
> the related ticket is about making DOIs optional

In digests. This expands it to all component DOIs.

File: src/api.raml
Comment (thewilkybarkid) [reply src/api.raml:464]: Want to say it was the latter. 🤔

**Comment by thewilkybarkid** [comment]
Can be merged when the implementation is done (Bot-Lax and/or Lax?).

**Comment by gnott** [comment]
I'm not sure exactly what is next. If I had to guess, don't we need the API to define `v3` articles before `lax` project can start serving them as `v3`, therefore this PR must be merged first?

**Comment by thewilkybarkid** [comment]
Meant ready rather than merged! But yeah, this would need to be merged first as Lax uses the schemas IIRC.

**Comment by lsh-0** [comment]
yup, that's all correct. Merge this first, then I can start work on updating lax. I'll probably have a load of questions as I pick through what the changes were.

**Comment by lsh-0** [comment]
v1 has been deprecated for a long time now. Can we/should we remove it in this PR?

**Comment by thewilkybarkid** [comment]
Should be able to remove it, but let's do that separately.

**Review by lsh-0 — APPROVED** [review]

**Comment by lsh-0** [comment]
Hi @thewilkybarkid , could you merge this if you're happy with it? I can then pin the revision in Lax and get the [tests passing](https://github.com/elifesciences/lax/pull/460). cheers :)

**Comment by lsh-0** [comment]
hey! thanks very much

**Pull request merged**
